collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -160,7 +160,20 @@ static enum cgroups_type cgroups_try_detect_version()
     if(!cgroups2_available)
         return CGROUPS_V1;
 
-    // 2. check systemd compiletime setting
+#if defined CGROUP2_SUPER_MAGIC
+    // 2. check filesystem type for the default mountpoint
```

Comment:
How about making it step 1? Any objections?

Reply:
ok, not a big deal, nvm then!

Reply:
I partly agree with you - there is no need to leave the check of filesystems. But if we want to refine the code, we should delete the whole detection part and make cgroups aware of hybrid configurations.